docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Controlling a WeDo 2.0 motor
+subject: Networking
+author: "@JorgePe"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+LEGO Education released the second version of WeDO in the beginning of 2016.
+We'll show how to use the bluez, the linux bluetooth stack, to wireless control a
+WeDo 2.0 motor.
+
+The first WeDO version uses USB so every robot needs to be tethered to a «host»
+(usually a computer but can also be a Mindstorms EV3 running ev3dev)
+The second WeDO version uses BLE (Bluetooth Low Energy, a sub-set of the Bluetooth
+4.0 standard) so robots can now be totally autonomous.
+
+## Requirements
+
+The EV3 internal bluetooth is to old, just 2.1 compliant. To use BLE we need an
+USB Bluetooth 4.0 dongle supported by ev3dev. If it works with Ubuntu or with a
```

Comment:
I read somehwere (probably here, https://en.wikipedia.org/wiki/Bluetooth_low_energy) that BLE was included in the BT 4.0 standard but it is optional to implement "classic" or "both" modes - thats strange for a standard. I tought that since most BT 4.0 smartphones and BT 4.0 USB dongles support BLE then no one was opting for just "classic".
LEGO might have made a bad decision with the newer EV3 (and strange decision since it's using BLE with the WeDO 2.0) but as BT 4.2 is pushing for even more "Smart" / BLE features it's just a matter of time to have BLE in the EV3.